src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -186,28 +210,45 @@ def get(self, request: Request, project: Project) -> Response:
                     outcome = munging_outcome
                     scope.set_tag("stacktrace_link.munged", True)
 
-                current_config = {"config": serialize(config, request.user), "outcome": outcome}
-                matched_code_mappings.append(current_config)
-                # use the provider key to be able to split up stacktrace
-                # link metrics by integration type
-                provider = current_config["config"]["provider"]["key"]
-                scope.set_tag("integration_provider", provider)  # e.g. github
+                current_config = {
+                    "config": serialize(config, request.user),
+                    "outcome": outcome,
+                    "automatically_generated": config.automatically_generated,
+                }
+                # Insert code mappings into the list in the order in which they are to be evaluated
+                index = self.get_matched_code_mapping_index(matched_code_mappings, current_config)
+                matched_code_mappings.insert(index, current_config)
 
-                if outcome.get("sourceUrl") and outcome["sourceUrl"]:
-                    result["sourceUrl"] = outcome["sourceUrl"]
-                    # if we found a match, we can break
-                    break
-
-            # Post-processing before exiting scope context
-            found: bool = result["sourceUrl"] is not None
-            scope.set_tag("stacktrace_link.found", found)
-            scope.set_tag("stacktrace_link.auto_derived", derived)
+            found: bool = False
+            derived = False
             if matched_code_mappings:
+                for code_mapping in matched_code_mappings:
```

Comment:
The "deleted" lines were moved here, as this is where we are iterating over the code mappings